specification/cognitiveservices/OpenAI.Inference/models/completions/extensions_onyourdata_vector_search.tsp

```diff
@@ -0,0 +1,63 @@
+import "@typespec/versioning";
+import "./extensions_onyourdata_authentication.tsp";
+
+namespace Azure.OpenAI;
+
+using TypeSpec.Versioning;
+
+@added(ServiceApiVersions.v2023_12_01_Preview)
+@doc("Embedding dependency types for vector search.")
+enum OnYourDataEmbeddingDependencyType {
+  @doc("Azure OpenAI Embedding endpoint URL.")
+  endpoint: "Endpoint",
+
+  @doc("Azure OpenAI Embedding deployment name.")
+  deploymentName: "DeploymentName",
```

Comment:
Are these referring to the endpoint of an Azure OpenAI resource that has an embeddings model deployed and the corresponding deployment name? If so, I would recommend mentioning so in the description.

Reply:
I'll start a conversation with the OYD team for this and the subsequent questions--agreed that we should clarify.

Reply:
Yes. Here is the longer version of the description: The Ada embedding model deployment name within the same Azure OpenAI resource. When this parameter is provided, Azure OpenAI on your data will use an internal call to evaluate the Ada embedding model, rather than calling the Azure OpenAI endpoint. This enables you to use vector search when the Azure OpenAI resource is in private networks.